apps/fabric-website/src/pages/HomePage/HomePage.module.scss

```diff
@@ -29,9 +29,9 @@
     background-position: center bottom;
     background-repeat: no-repeat;
     background-size: contain;
-    bottom: -10px;
+    bottom: -3px;
```

Comment:
These numbers feel a bit random, i would either create variables for what this and number below are or i would see if there is a way to reshape the containing elements so we don't have to pull this down -3px.

Reply:
@battletoilet Yes, I was pixel pushing to get the hero and angle svgs to span the app, given I had to increase the width of the app to match UHF. I am all for reshaping in another way - let's sync up on that today.

Reply:
For now let's just store this value in a variable with a name that explains it a bit. We can always add this to the backlog and get to it later.